Managers/SceneManager.gd

```diff
@@ -75,3 +79,38 @@ func on_defeat() -> void:
 ## Give rewards, allow player to move on the map
 func on_event_win() -> void:
 	PlayerManager.player_room.room_event.on_event_ended()
+
+func _save_scene_data() -> void:
+	for children: Node in current_scene.get_children():
+		children.owner = current_scene
```

Comment:
what is the goal of this ?

Reply:
Was reading a thread on saving a PackedScene and it mentioned to set the children's owner to the current scene. However, I removed this for loop and it seems to still work, so I'm removing it!

Reply:
alright
if you still remember where you read that I would be curious to see, if you have a link